src/lib/fillEvent.ts

```diff
@@ -0,0 +1,102 @@
+import { knexController } from '../database/database';
+import {
+  FillEventResponse,
+  GasPrices,
+  GasPressures,
+} from '../types/fillEvent.types';
+
+export const getGasPrice = async (gas: string): Promise<number | undefined> => {
+  const res = await knexController('prices')
+    .orderBy('created_at', 'desc')
+    .first('price_per_litre_in_eur_cents as price')
+    .where({ gas });
+  if (res === undefined) return undefined;
+  return res.price;
+};
+
+export const getGasPrices = async (): Promise<GasPrices | undefined> => {
+  const oxygen = await getGasPrice('oxygen');
+  const helium = await getGasPrice('helium');
+  const argon = await getGasPrice('argon');
+  const diluent = await getGasPrice('diluent');
```

Comment:
Jos ei haeta kaikkia kaasuja yhdellä haulla, niin nää pitäis ainakin laittaa transactionin sisään